wrappers/python/tests/layer/test_llama_attention.py

```diff
@@ -384,3 +384,43 @@ def test_llama_attn_kvcache(
     nntile_layer.unregister()
     nntile_layer.x.unregister()
     nntile_layer.y.unregister()
+
+
+@pytest.mark.benchmark
+def test_bench_llama_attention_forward_async(context_cuda, benchmark_operation):
+    dtype = 'fp32'
+    params = single_tile
+    bias = False
+    flash_attention = False
+    _, nntile_layer, *_ = generate_inputs(dtype, params, bias, flash_attention)
+
+    nntile_layer.clear_gradients()
+    out_np = np.zeros(nntile_layer.y.value.shape, dtype=np.float32, order='F')
+
+    def bench_fn():
+        nntile_layer.forward_async()
+        nntile_layer.y.value.to_array(out_np)
+
+    nntile.starpu.wait_for_all()
+    benchmark_operation(bench_fn)
+
+
+@pytest.mark.benchmark
+def test_bench_llama_attention_backward_async(context_cuda, benchmark_operation):
+    dtype = 'fp32'
+    params = single_tile
+    bias = False
+    flash_attention = False
+    _, nntile_layer, *_ = generate_inputs(dtype, params, bias, flash_attention)
+
+    nntile_layer.clear_gradients()
+    rng = np.random.default_rng(42)
+    grad_np = np.array(rng.standard_normal(nntile_layer.y.value.shape), dtype=np.float32, order='F')
+
+    def bench_fn():
+        nntile_layer.forward_async()
+        nntile_layer.y.grad.from_array(grad_np)
```

Comment:
If my memory does not fail me, `y.grad` is NOT invalidated during `backward_async`. It shall not. Therefore, set it up only once before the benchmark.